packages/m2m-gateway/src/services/tenantService.ts

```diff
@@ -110,5 +128,25 @@ export function tenantServiceBuilder(clients: PagoPAInteropBeClients) {
         },
       };
     },
+    async addCertifiedAttribute(
+      tenantId: TenantId,
+      seed: m2mGatewayApi.TenantCertifiedAttributeSeed,
+      { logger, headers }: WithLogger<M2MGatewayAppContext>
+    ): Promise<void> {
+      logger.info(
+        `Assigning certified attribute ${seed.id} to tenant ${tenantId}`
+      );
+
+      const response =
+        await clients.tenantProcessClient.tenantAttribute.addCertifiedAttribute(
```

Comment:
Linking the previous unaddressed review comment here, as it was hard to see https://github.com/pagopa/interop-be-monorepo/pull/1819#pullrequestreview-2832574919

Reply:
Sorry, didn't notice!